techniques/css/C42.html

```diff
@@ -181,20 +179,20 @@ <h4>The CSS</h4>
 
 
 <section id="tests">
-			<h2>Tests</h2>
+			<h2>検証</h2>
 			
 			<section class="test-procedure" id="procedure">
-				<h3>Procedure</h3>
+				<h3>手順</h3>
 
-				<p>For each target that cannot be enlarged by a mechanism, is not inline, and is not covered by the essential exception:</p>
+				<p>メカニズムによって拡大できず、インラインでなく、必要不可欠の例外に該当しない各ターゲットに対して:</p>
 				<ul>
-					<li>Check that the target has enough spacing so that the space around it measures at least 44px width and 44px height.</li>
+					<li>ターゲットの周囲のスペースが少なくとも幅 44 px、高さ 44 px となるように、十分な間隔が確保されていることをチェックする。</li>
 				</ul>
 			</section>
 			<section class="test-results" id="expected-results">
-				<h3>Expected Results</h3>
+				<h3>期待される結果</h3>
 				<ul>
-					<li>Check #1 is true</li>
+					<li>チェック 1 が真である。</li>
```

Comment:
ここの表現ですが、既存の他のテクニック (WCAG 2.1 達成方法集) では「#1 及び #2 の結果が真である。」のような表現になっています。
(とはいえ、ここでは、確認項目が一つしかなく、特にナンバリングされているわけではないんですよね...)

```suggestion
					<li>#1 の結果が真である。</li>
```